.gitignore

```diff
@@ -1,5 +1,5 @@
 coverage
-node_modules
+/node_modules
```

Comment:
Yes. One of the test needs to have a mock `node_modules` dir within test dir, so it should not be ignored, only root one should be.